packages/css/avatar.css

```diff
@@ -0,0 +1,93 @@
+.ds-avatar {
+  --ds-avatar-background: var(--ds-color-accent-base-default);
+  --ds-avatar-color: var(--ds-color-accent-contrast-default);
+  --ds-avatar-size: var(--ds-sizing-12);
+  --ds-avatar-padding: var(--ds-spacing-2);
+  --ds-avatar-border-radius: 50%;
+
+  background-color: var(--ds-avatar-background);
+  height: var(--ds-avatar-size);
+  aspect-ratio: 1 / 1;
+  color: var(--ds-avatar-color);
+  border-radius: var(--ds-avatar-border-radius);
+  overflow: hidden;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  user-select: none;
+  text-transform: uppercase;
+  text-decoration: none;
+
+  &:has(> svg) {
+    padding: var(--ds-avatar-padding);
+  }
+
+  & img {
+    object-fit: cover;
+    width: 100%;
+    height: 100%;
+  }
+
+  &:empty::before {
+    content: '';
+    width: calc(100% - (var(--ds-avatar-padding) * 2));
+    height: calc(100% - (var(--ds-avatar-padding) * 2));
+    mask-repeat: no-repeat;
+    mask-size: contain;
+    background-color: var(--ds-avatar-color);
+    mask-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='1em' height='1em' fill='none' viewBox='0 0 24 24' focusable='false' role='img'%3E%3Cpath fill='currentColor' fill-rule='evenodd' d='M8.25 7.5a3.75 3.75 0 1 1 7.5 0 3.75 3.75 0 0 1-7.5 0M12 2.25a5.25 5.25 0 1 0 0 10.5 5.25 5.25 0 0 0 0-10.5M8.288 17.288A5.25 5.25 0 0 1 17.25 21a.75.75 0 0 0 1.5 0 6.75 6.75 0 0 0-13.5 0 .75.75 0 0 0 1.5 0 5.25 5.25 0 0 1 1.538-3.712' clip-rule='evenodd'%3E%3C/path%3E%3C/svg%3E");
+  }
+
+  &[data-ds-variant='circle'] {
+    --ds-avatar-border-radius: 50%;
```

Comment:
We dont have a `var(--ds-border-radius-full)` yet right? 🤔

Reply:
Right, we do 😅